example/build.gradle

```diff
@@ -14,12 +20,15 @@ android {
         versionName "1.0"
 
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
+
+        buildConfigField "String", "API_KEY", "\"" + (localProperties['cloud.eppo.apiKey'] ?: "need to set cloud.eppo.apiKey in local.properties") + "\""
```

Comment:
make `cloud.eppo.apiKey` in `local.properties` available as `API_KEY` to the example application